### ISSUE TITLE
NVDA not reading Thunderbird Add-Ons menu items (Add-on names)

### ISSUE THREAD

**Issue by britechguy** [nvaccess/nvda#10957]
### Steps to reproduce:
Open Thunderbird
Use ALT+T,O to open the Tools Menu, Add-on Options item
Hit Enter to get list of installed add ons.
Down arrow through add-ons list and NVDA is silent, not reading the add-on name

NOTE:  This behavior is not 100% consistent, when trying again just now, with only two add-ons, Quick Folder Key Navigation, and TbSync, on one try it read the first and not the second, then once the second but not the first, but after having read either one, only once, all arrowing among the list items produces dead silence.

### Expected behavior:
At down arrowing stage the names of the Add-ons should be announced as the list is traversed, every time it is traversed, in either direction.

**Comment by Adriani90** [comment]
Please run the com registration fixing tool and then try to reproduce again

**Comment by britechguy** [comment]
Ran the COM Registration Fixing Tool and restarted my computer.

All appears to be working fine now.  I'm actually surprised, and pleasantly, as when I've needed to run that tool in the past there was a lot more "out of whack" with NVDA.  I'll remember to give it a shot in the future with more subtle issues first, before submitting an issue.

By the way, is the issue template available in some form for editing in something like Word or Notepad, where you could cut and paste the completed form straight into the Bug Report or Feature Suggestion (I think that's what the other button says, but not having that screen in front of me right now I am not sure.  I imagine the template for that is different than an issue report as well).

**Comment by Brian1Gaff** [comment]
I tend to go to the new issue and cut and paste the text to an editor there, 
then come out without creating the report. You are right its far easier to 
do it off  line, no latency effects from the site, either, then all you have 
to do is remember to do a complete delete on the main field before grabbing 
and pasting in your finished report .
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
-----

**Comment by Adriani90** [comment]
Thanks for reporting. Closing as works for me.
Yes, the template for a feature is different from this one. You have to login first, then press "new issue" and then you can choose wether you want to create a bug or a feature.
